### PR TITLE
feature/Task3AddhyperlinktoLoginPage

### PR DESCRIPTION
**Pull request by ROAHIT**

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -12,11 +12,46 @@
         h1
```
Comment (bharathreo29) [inline homepage.html:12]: Put all the style code into a "Style" folder and make reference from there. Study about the methods to implement css codes.

File: homepage.html [hunk 2]
```diff
@@ -12,11 +12,46 @@
...
         {
             position: relative;
-            top: 50%;
+            top: 30%;
```
Comment (BharathCode1) [inline homepage.html:15]: @ROAHIT : Put all the style code into a "Style" folder and make reference from there. Study about the methods to implement css codes.

File: homepage.html [hunk 3]
```diff
@@ -12,11 +12,46 @@
...
+        {
+            font-size: 30px;
+        }
     </style>
     <body>
-        <h1>WELCOME</h1>
+        <h1>WELCOME!</h1>
+        <p><a href="http://127.0.0.1:5500/loginlink.html"><button>LOGIN</button></a></p>
+        <div>
+            <script type="text/javascript">
```
Comment (BharathCode1) [inline homepage.html:39]: @ROAHIT : Put javascript code in a separate folder and refer the script from there. Study about the methods to invoke javascript code inside a html file.

**Review by BharathCode1 — CHANGES_REQUESTED** [review]

**Comment by ROAHIT** [comment]
created another feature branch from a branch instead of main branch

**Pull request closed without merging**